novatel_gps_driver/src/nodelets/novatel_gps_nodelet.cpp

```diff
@@ -168,7 +171,12 @@ namespace novatel_gps_driver
   class NovatelGpsNodelet : public nodelet::Nodelet
   {
   public:
-    NovatelGpsNodelet() :
+      uint8_t status_gps;
+      ros::Time last_update_time_;
+      cav_msgs::DriverStatus status_;
```

Comment:
If you are going to us _ at the end of member variable names you should be consistent. Please add a _ to the end of the other member variables you added